lib/components/NodeBaseInputField.tsx

```diff
@@ -1,101 +1,150 @@
-import { CSSProperties, memo, ReactNode, useRef, useState } from 'react'
+import { CSSProperties, ReactNode, useRef, useState } from 'react'
 import { NodeInputConfig } from '../config'
 
-type NodeBaseInputFieldProps = Pick<NodeInputConfig, 'valueType' | 'name'> & {
+export type InputHTMLTypes =
+  | 'checkbox'
+  | 'color'
+  | 'date'
+  | 'datetime-local'
+  | 'email'
+  | 'month'
+  | 'number'
+  | 'password'
+  | 'range'
+  | 'tel'
+  | 'text'
+  | 'time'
+  | 'url'
+  | 'week'
```

Comment:
Can we use `React.HTMLInputTypeAttribute` instead?

Reply:
Sure. I'll Omit the the one like button, reset, submit, that don't make sense. Or maybe not bother?

Reply:
Good catch, sounds like we only support a subset. If that's true then yeah I think an `Omit<...>` makes good sense if you're comfortable with that as well.